### DISCUSSION TITLE
firefly 2.0.3 update

### DISCUSSION THREAD

**Discussion by neutron54** [iotaledger/firefly#7704] [Help & Support]
Hello, I recently  updated my iota wallet to 2.0.3. Before the update there were 2998 iotas in my wallet , but now I can see only  0.52. what's the problem ? what can i do for it?

**Comment by begonaalvarezd** [comment]
Run the balance finder from settings > advanced > balance finder, until all your funds appear, you need to allow the new firefly to re-descover all your funds spread across all your addresses
